packages/utxo-lib/tests/coinselect/utils.test.ts

```diff
@@ -1,28 +1,36 @@
-import { uintOrNaN, bignumberOrNaN, getFee } from '../../src/coinselect/utils';
+import { formatStringFeeRate, bignumberOrNaN, getFee } from '../../src/coinselect/utils';
 
 describe('coinselect utils', () => {
-    it('uintOrNaN', () => {
-        expect(uintOrNaN(1)).toEqual(1);
-        expect(uintOrNaN(Infinity)).toBeUndefined();
-        expect(uintOrNaN(NaN)).toBeUndefined();
-        expect(uintOrNaN(1.1)).toBeUndefined();
-        expect(uintOrNaN(-1)).toBeUndefined();
+    it('formatStringFeeRate', () => {
+        expect(formatStringFeeRate('1')).toEqual(1);
+        expect(formatStringFeeRate('1.1')).toEqual(1.1);
+        expect(formatStringFeeRate('-1')).toBeUndefined();
+        expect(formatStringFeeRate('aaa')).toBeUndefined();
+        expect(formatStringFeeRate('')).toBeUndefined();
         // @ts-expect-error invalid arg
-        expect(uintOrNaN('')).toBeUndefined();
+        expect(formatStringFeeRate(1)).toBeUndefined();
         // @ts-expect-error invalid arg
-        expect(uintOrNaN('1')).toBeUndefined();
+        expect(formatStringFeeRate(1.1)).toBeUndefined();
         // @ts-expect-error invalid arg
-        expect(uintOrNaN()).toBeUndefined();
+        expect(formatStringFeeRate(-1)).toBeUndefined();
+        // @ts-expect-error invalid arg
+        expect(formatStringFeeRate()).toBeUndefined();
+        // @ts-expect-error invalid arg
+        expect(formatStringFeeRate(null)).toBeUndefined();
+        // @ts-expect-error invalid arg
+        expect(formatStringFeeRate(NaN)).toBeUndefined();
+        // @ts-expect-error invalid arg
+        expect(formatStringFeeRate(Infinity)).toBeUndefined();
     });
 
     it('bignumberOrNaN', () => {
         expect(bignumberOrNaN('1')).not.toBeUndefined();
-        expect(bignumberOrNaN('1.1')).toBeUndefined();
+        // expect(bignumberOrNaN('1.1')).toBeUndefined();
```

Comment:
why commented? those should be still valid (invalid) cases, no?

Reply:
A typo somehow... Fixed 463ab037d9c46a43792a5ad5b0e6f993c4cf7d1d